src/argilla/server/apis/v0/handlers/datasets.py

```diff
@@ -28,6 +27,7 @@
 from argilla.server.schemas.v0.datasets import CopyDatasetRequest, CreateDatasetRequest, Dataset, UpdateDatasetRequest
 from argilla.server.security import auth
 from argilla.server.services.datasets import DatasetsService
+from argilla.utils.pydantic import parse_obj_as
```

Comment:
Even if the client and server are sharing this new behavior, I would duplicate the new pydantic.py module for both, client and server separately, so in the future we can separate both packages